packages/react-magma-dom/src/components/Input/Input.stories.tsx

```diff
@@ -38,8 +43,9 @@ export const Default = Template.bind({});
 Default.args = {
   disabled: false,
   helperMessage: 'Helper message',
-  placeholder: 'Placeholder text...'
+  placeholder: 'Placeholder text...',
 };
+Default.parameters = { controls: { exclude: ['iconPosition'] } };
```

Comment:
Any Input Storybook Story that does not have the icon defined should have this line:
- Error
- Large
- File
- Inverse